webgl-obj-loader.js

```diff
@@ -188,10 +192,18 @@
                   unpacked.textures.push(+textures[(vertex[1] - 1) * 2 + 0]);
                   unpacked.textures.push(+textures[(vertex[1] - 1) * 2 + 1]);
                 }
-                // vertex normals
-                unpacked.norms.push(+vertNormals[(vertex[2] - 1) * 3 + 0]);
-                unpacked.norms.push(+vertNormals[(vertex[2] - 1) * 3 + 1]);
-                unpacked.norms.push(+vertNormals[(vertex[2] - 1) * 3 + 2]);
+                // Vertex normals when format f v/vn
+                else if (vertex.length === 2) {
+                  unpacked.norms.push(+vertNormals[(vertex[1] - 1) * 3 + 0]);
+                  unpacked.norms.push(+vertNormals[(vertex[1] - 1) * 3 + 1]);
+                  unpacked.norms.push(+vertNormals[(vertex[1] - 1) * 3 + 2]);
+                }
+                // Vertex normals when format 'f v//vn' or 'f v/vt/vn'
+                if (vertex.normals === 3) {
```

Comment:
I'm assuming `vertext.normals` here should be `vertex.length`?

Also, this should probably be an `else if` since, logically, it belongs with the checks above.